eng/common/scripts/stress-testing/stress-test-deployment-lib.ps1

```diff
@@ -154,7 +154,10 @@ function DeployStressPackage(
     if ($repositoryBase) {
         $imageTag += "/$repositoryBase"
     }
-    $imageTag += "/$($pkg.Namespace)/$($pkg.ReleaseName):${deployId}"
+    if ($pkg.Namespace -ne $repositoryBase) {
```

Comment:
I think it would be better to fix the issue at the root in find-all-stress-packages since that's where the context of how we should set these values lives (i.e. whether to use the username or the chart metadata). Perhaps exposing another property like `$pkg.ImageRepository` and setting that alongside the other "is local user mode" conditionals could work?